lib/utils.js

```diff
@@ -171,24 +172,11 @@ async function parseLocalizableStrings (opts) {
   }
 }
 
-async function getBundleIdFromApp (app) {
-  logger.debug("Getting bundle ID from app");
```

Comment:
Can you add a log line that would replace this. It would be good to have the bundleId in the log message, too, for posterity.